plugins/unicodelib/unicodelib.cpp

```diff
@@ -1767,6 +1768,26 @@ UNICODELIB_API unsigned UNICODELIB_CALL ulUnicodeLocaleEditDistance(unsigned lef
 }
 
 
+UNICODELIB_API unsigned UNICODELIB_CALL ulUnicodeLocaleEditDistanceV2(unsigned leftLen, UChar const * left, unsigned rightLen, UChar const * right, char const * localename, unsigned radius)
+{
+    BreakIterator* bi = 0;
```

Comment:
Should existing function ulUnicodeLocaleEditDistance  call on to this one (passing 0 for radius) ?

Reply:
changed